src/utils/StringUtils.ts

```diff
@@ -1,11 +1,17 @@
+/**
+ * TODO v3.0.0
+ * Move all string-related utility functions in /src/utils to this file.
```

Comment:
I'm not sure this is actually a goal. "*Utils" files are something of an anti-pattern, they tend to collect lots of unrelated functionality without rhyme or reason. In our case, this can be problematic for dependency-related reasons. I'm pretty sure Snarling was trying to shard functions into smaller files, not larger ones.

Reply:
I partially disagree. X-Utils files are okay when they group "related" functions. The problem is how we decide which things are "related". For example, it's fine to group `createRandomString` (`src\utils\helpers\createRandomString.ts`), `trimQuotes` (`src\utils\helpers\string.ts`), `generateRandomString`, `containsAllStrings`, `longestCommonStart`, etc. IMO, "one function per file" (many cases in `src\utils\helpers`) is only good for a "utility" library (like lodash). The concern of "dependency hell" is valid, though.

I'll still revert the change in the filename and the new note. It's best to discuss and implement it in another PR.

Reply:
I should clarify my position a bit more, which is: I'm not sure I have a position. XD

Specifically, *Utils files are dangerous because they *tend to* accumulate unrelated functions. It doesn't mean that they're *wrong* though, and sometimes even a dangerous pattern is still the right choice. 

The main thing I was objecting to was the TODO, because I'm not sure that we want to go that direction overall. Maybe we do, but it seemed contrary to what I've seen Snarling do. The renaming you did here seemed fine.